Lib/fontbakery/profiles/googlefonts.py

```diff
@@ -123,6 +123,7 @@
    'com.google.fonts/check/integer_ppem_if_hinted',
    'com.google.fonts/check/unitsperem_strict',
    'com.google.fonts/check/contour_count',
+   'com.google.fonts/check/vertical_metrics_regressions',
```

Comment:
do you think we're likely to have a few more regression checks? If so, then we could call them all `com.google.fonts/check/regression/something`. What do you think?

Reply:
Can we make this change if we add more regressions checks? I can't think of any off the top of my head that I would like to add. On the other hand, we already have a missing codepoints check.

Reply:
we can change this later. Prior to Fontbakery 1.0.0 we are still being flexible on renaming check-IDs